frontend/src/pages/exploreApplication/ExploreApplications.tsx

```diff
@@ -20,7 +19,7 @@ import { fireTrackingEvent } from '../../utilities/segmentIOUtils';
 import { ODH_PRODUCT_NAME } from '../../utilities/const';
 import { useAppContext } from '../../app/AppContext';
 
-import './ExploreApplications.scss';
+import 'DrawerContentBody.scss';
```

Comment:
npm build is failing due to this requiring explicitly referencing this module exists in the same directory

```suggestion
import './DrawerContentBody.scss';
```